### PR TITLE
Version 4.4.4 to Develop

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "name": "event-tickets",
+<<<<<<< HEAD
```
Comment (barryhughes) [inline package.json]: Needs resolution

Comment (bordoni) [inline package.json]: Thx for catching that

Comment (bordoni) [inline package.json]: _Resolved_

**Comment by barryhughes** [comment]
:smiley:

**Pull request merged**